README.md

```diff
@@ -32,7 +32,7 @@ ext install idbartosz.darkpp-italic
     }
     ```
 3. Tweek font appearance by editing VScode internal styles *(optional step)*.
-    1. Find in your OS `<path to VScode>\resources\app\out\vs\workbench.main.css`
+    1. Find in your OS `<path to VScode>/resources/app/ut/vs/workbench/workbench.desktop.main.css`
```

Comment:
First of all sorry for my slow response. Yes you're right, it seems that with latest VSCode they've changed naming.

Thank you, I'll merge your PR when you fix "typo" and update [changelog](https://github.com/idbartosz/vscode-darkpp-italic/blob/master/CHANGELOG.md).
```suggestion
    1. Find in your OS `<path to VScode>/resources/app/out/vs/workbench/workbench.desktop.main.css`
```